src/app/applications/[id]/page.tsx

```diff
@@ -44,6 +46,10 @@ export default function ApplicationDetailsPage() {
     }
   }, [error]);
 
+  if (errorStatusCode === 404) {
```

Comment:
@brunopacheco1 the backend throws 500 for wrong application id at the moment. You need to create a ticket to debug that

<img width="365" alt="Screenshot 2024-07-05 at 15 58 25" src="https://github.com/GenomicDataInfrastructure/gdi-userportal-frontend/assets/1210046/1175f7ec-5f34-4e28-b671-ff45aaea261c">